README.md

```diff
@@ -58,12 +58,20 @@ In `src/config.js` multiple config variables are available for tweaking. Copy
 for your own bot rather than directly modifying `src/config.js` defaults.
 
 ## Upload
+### install dependencies
 
     npm install
 
+### add your account credentials
+#### via env
     export email=EMAIL
     export password=PASSWORD
 
+#### via git ignored file
+    echo "module.exports = { email: 'your-email@here.tld', password: 'your-secret' };" > account.screeps.com.js
```

Comment:
sorry, i was to fast writeing docs... WRONG PATH
please update with the following:
echo "module.exports = { email: 'your-email@here.tld', password: 'your-secret' };" > ./src/account.screeps.com.js
(after sitting in the root: screeps.com folder)